core/src/main/resources/schema/sqlserver/sqlserver-create-schema.sql

```diff
@@ -40,3 +40,24 @@ CREATE TABLE "snapshot" (
     PRIMARY KEY ("persistence_id", "sequence_number")
   )
 
+-- Create Sequence Object
+CREATE SEQUENCE global_offset
+    START WITH 1
+    INCREMENT BY 1;
+
+CREATE TABLE durable_state
+(
+    "global_offset"         BIGINT
+        CONSTRAINT [df_global_offset] DEFAULT
+        (NEXT VALUE FOR global_offset),
+    "persistence_id"        VARCHAR(255)   NOT NULL,
+    "revision"              NUMERIC(10, 0) NOT NULL,
```

Comment:
why not use bigint, because pg use bigint

Reply:
Just to maintain the same column type as Event Journal.